utils/symbol-rawelf.h

```diff
@@ -91,7 +92,7 @@ struct uftrace_elf_iter {
 #define elf_for_each_shdr(elf, iter)					\
 	for (elf_get_strtab((elf), (iter), (elf)->ehdr.e_shstrndx),	\
 		     (iter)->i = 0, (iter)->nr = (elf)->ehdr.e_shnum;	\
-	     (iter)->i < (iter)->nr &&					\
+		     (iter)->i < (iter)->nr && (elf)->have_section &&	\
```

Comment:
Please don't change the indentation - it's there to identify the start of second part of the for statement.